Skia build is broken

@kjlubick, could you please take a look at https://oss-fuzz-build-logs.storage.googleapis.com/index.html

There are lots of compilation errors due to C-style type casts when calling intrinsic functions:

```
<...>
Step #3: ../../src/core/../opts/SkNx_sse.h:76:43: error: cast from 'const void *' to 'float *' drops const qualifier [-Werror,-Wcast-qual]
Step #3:         __m128 v0 = _mm_loadu_ps(((float*)ptr) +  0),
Step #3:                                           ^
<...>
```

--- a/projects/skia/build.sh
+++ b/projects/skia/build.sh
@@ -26,7 +26,7 @@ $SRC/depot_tools/gn gen out/Fuzz\
     cxx="'$CXX'"
     is_debug=false
     extra_cflags=["'"$CXXFLAGS_ARR"'","-DIS_FUZZING",
-        "-Wno-zero-as-null-pointer-constant", "-Wno-unused-template"]
+        "-Wno-zero-as-null-pointer-constant", "-Wno-unused-template", "-Wno-cast-qual"]
     skia_use_system_freetype2=false
     skia_use_fontconfig=false
     skia_enable_gpu=false

